public/index.html

```diff
@@ -8,6 +8,45 @@
   <title>ROAR!</title>
 </head>
 <body>
-  <h1 class="title">ROAR!</h1>
+  <h1 class="title">Bring attention to problems on the web</h1>
+  <div class="steps">
+    <div class="step">
```

Comment:
If we decide to use semantic tags, and I believe we should, the "step" `div`s should be `section`s.

Reply:
resolved